app/agents/voice/automatic/processors/llm_spy.py

```
@@ -25,6 +25,7 @@
 from pipecat.utils.tracing.turn_context_provider import (
     get_current_turn_context,
 )
+from pipecat.utils.tracing.turn_context_provider import get_current_turn_context
```

Ruff (0.14.5)

Redefinition of unused `get_current_turn_context` from line 26

Remove definition: `get_current_turn_context`

(F811)